include/af/vision.h

```diff
@@ -552,6 +609,41 @@ extern "C" {
     AFAPI af_err af_dog(af_array *out, const af_array in, const int radius1, const int radius2);
 #endif
 
+#if AF_API_VERSION >= 32
+    /**
+       C Interface wrapper for Homography estimation
+
+       \param[out] H is a 3x3 array containing the estimated homography.
+       \param[out] inliers is the number of inliers that the homography was estimated to comprise,
+                   in the case that htype is AF_HOMOGRAPHY_RANSAC, a higher inlier_thr value will increase the
+                   estimated inliers. Note that if the number of inliers is too low, it is likely
+                   that a bad homography will be returned.
+       \param[in]  x_src x coordinates of the source points.
+       \param[in]  y_src y coordinates of the source points.
+       \param[in]  x_dst x coordinates of the destination points.
+       \param[in]  y_dst y coordinates of the destination points.
+       \param[in]  inlier_thr if htype is AF_HOMOGRAPHY_RANSAC, this parameter will five the maximum L2-distance
+                   for a point to be considered an inlier.
+       \param[in]  iterations maximum number of iterations when htype is AF_HOMOGRAPHY_RANSAC and backend is CPU,
+                   if backend is CUDA or OpenCL, iterations is the total number of iterations, an
+                   iteration is a selection of 4 random points for which the homography is estimated
+                   and evaluated for number of inliers.
+       \param[in]  af_homography_type can be AF_HOMOGRAPHY_RANSAC, for which a RANdom SAmple Consensus will be
+                   used to evaluate the homography quality (e.g., number of inliers), or AF_HOMOGRAPHY_LMEDS,
+                   which will use Least Median of Squares method to evaluate homography quality.
+       \param[in]  dtype the array type for the homography output.
+       \param[out] out is difference of smoothed inputs.
+       \return     \ref AF_SUCCESS if the computation is is successful,
```

Comment:
* `out` is not a parameter name
* flip the order of `htype`, `inlier_thr` and `iterations` to match the function parameter order
* change `af_homography_type` to `htype` in the documentation
* change `type/dtype` to `otype`